sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Remove requires io.netty.transport.epoll from module-info.
+* Converted from `durationInMicroSec` to `durationInMilliSecs` in `CosmosDiagnostics` - See [PR 29643](https://github.com/Azure/azure-sdk-for-java/pull/29643)
```

Comment:
Please remove `Converted from `durationInMicroSec` to `durationInMilliSecs` in `CosmosDiagnostics` - See [PR 29643](https://github.com/Azure/azure-sdk-for-java/pull/29643)`. - this change has already been released